save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt

```diff
@@ -326,6 +337,35 @@ class SandboxController(
             }
     }
 
+    /**
+     * @param authentication
+     * @return logs from agent sandbox
+     */
+    @GetMapping("/logs-from-agent")
+    fun getAgentLogs(
+        authentication: Authentication,
+    ): Flux<String> = blockingToMono {
+        sandboxExecutionRepository.findTopByUserIdOrderByStartTimeDesc(authentication.userId())
```

Comment:
better to migrate debugInfo to executionId too

Reply:
it's a big change -- now `userId` is main key for all objects in sandbox.
we can add checking by last execution... but it's not required I guess.